### ISSUE TITLE
On-demand speech mode - NVDA still talks in PowerPoint when going to next slide

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#16825]
### Steps to reproduce:
* Enable on-demand speech mode
* Open a PowerPoint file
* Press F5 to start the slideshow
* Press space to jump to next slide

### Actual behavior:
NVDA reports the slide content
### Expected behavior:
NVDA should not speak since the action to open the slideshow (F5) or to go to next slide (space) should be executable without any speech feedback in on-demand speech mode.
### NVDA logs, crash dumps and other attachments:
N/A
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2024.2beta3
#### Windows version:
Windows 10 22H2 (AMD64) build 19045.4529
#### Name and version of other software in use when reproducing the issue:
PowerPoint, Office 2016
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No, but this issue is probably here since on-emand speech mode was implemented.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
N/A

**Comment by CyrilleB79** [comment]
Note: this also happens in browsers when the page is loaded or in Outlook when opening a message.
